lib/services/internalEngine/index.js

```diff
@@ -79,10 +79,19 @@ class InternalEngine {
     this.bootstrap = bootstrap;
 
     if (!this.client) {
-      this.client = new es.Client({
-        hosts: this.config.hosts || this.config.host + ':' + this.config.port,
-        apiVersion: this.config.apiVersion
-      });
+      let options;
+
+      if (this.config.client) {
+        options = Object.assign({}, this.config.client);
+      }
+      else {
+        options = {
```

Comment:
nipticking: add here the same comment as https://github.com/kuzzleio/kuzzle/blob/68a17b30276dd2b1c04f3ae751fe53fcef014a32/lib/services/elasticsearch.js#L1189